ush/wrappers/cheyenne/get_lbcs.qsub

```diff
@@ -0,0 +1,18 @@
+#! /bin/sh
+#PBS -A epic
+#PBS -l select=1:mpiprocs=1:ncpus=1:mem=2G
+#PBS -q regular
+#PBS -l walltime=01:30:00
+#PBS -j oe -o get_extrn_lbcs.log
+#PBS -N get_extrn_lbcs
+
+export GLOBAL_VAR_DEFNS_FP=$PWD'/../var_defns.sh'
+export USHdir=$PWD'/../../../ufs-srweather-app/ush'
```

Comment:
suggesting 
export USHdir=$PWD'/../../ush'

same as in 
https://github.com/ufs-community/ufs-srweather-app/pull/924/files#r1357370546

Reply:
If this is OK to use 
USHdir=$PWD'/../../ush'
that would be the same change for all the scripts. I'd be happy to help with the bulk editing (as a PR to https://github.com/RatkoVasic-NOAA/ufs-srweather-app/tree/wrappers maybe?) Users may use an upper-level SWR directory named differently from ufs-srweather-app 


On the other hand, level of details used in the existing version of 
`export USHdir=$PWD'/../../../ufs-srweather-app/ush'`
provides more indication of where the ./ush directory is located in relation to the upper-level SRW (ufs-srweather-app)

Either way would be OK with me, just suggesting options to improve